src/test/java/ca/qaguru/oranghrmbatch27/tests/CustomersTests.java

```diff
@@ -0,0 +1,73 @@
+package ca.qaguru.oranghrmbatch27.tests;
+
+import ca.qaguru.oranghrmbatch27.library.TestBase;
+import ca.qaguru.oranghrmbatch27.pages.CustomersPage;
+import ca.qaguru.oranghrmbatch27.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CustomersTests extends TestBase {
+
+    //BATCH29-10
+    @Test
+    public void addNewCustomer()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin","admin123",true,null);
+        CustomersPage customersPage = new CustomersPage(driver);
+        customersPage.navigateToCustomer();
+        Boolean status = customersPage.addCustomer("Anoop Joseph","Flexible pouch manufacturer");
```

Comment:
The data can be included in separate string variables

Reply:
Created separate  varibles to store the information